packages/@aws-cdk/tmp-toolkit-helpers/src/api/diff/diff-formatter.ts

```diff
@@ -122,10 +122,8 @@ export interface TemplateInfo {
 
   /**
    * The name of the stack
-   *
-   * @default undefined
    */
-  readonly stackName?: string;
+  readonly stackName: string;
```

Comment:
Breaking change? Not sure if it matters, though.

Reply:
diff-formatter isn't public so this isn't breaking. i made this change because we do always have the stackName, since we have the stack